### PR TITLE
CIRCSTORE-534: Release 17.3.0

### PR DESCRIPTION
**Pull request by imerabishvili**
Purpose:
[CIRCSTORE-534](https://folio-org.atlassian.net/browse/CIRCSTORE-534)

### CODE REVIEW

**Review by AntonAntonich — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
-## 17.3.0
-* [CIRCSTORE-533](https://folio-org.atlassian.net/browse/CIRCSTORE-533) Upgrade "holdings-storage" to 8.0
+## 17.3.0 2024-10-30
+* Upgrade RMB to 35.3.0, Vert.x to 4.5.9 (CIRCSTORE-520, CIRCSTORE-536)
+* Update module permissions in the ModuleDescriptor (CIRCSTORE-533)
+* Add checkout slips template (CIRCSTORE-529)
+* Setting name should be unique (CIRCSTORE-519)
+* Merge settings API into master branch (CIRCSTORE-515)
+* Get the number of copies and the last print details (CIRCSTORE-514)
+* Create print_events_log table to persists the pickslips print event logs. (CIRCSTORE-513)
```
Comment (roman-barannyk) [inline NEWS.md:8]:
```suggestion
* Create `print_events_log` table to persists the pickslips print event logs (CIRCSTORE-513)
```

File: NEWS.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+* Create a new settings table in circ-store to persist the values needed for circ modules. (CIRCSTORE-512)
```
Comment (roman-barannyk) [inline NEWS.md:9]:
```suggestion
* Create a new settings table in circ-store to persist the values needed for circ modules (CIRCSTORE-512)
```

File: NEWS.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+* API version update (CIRCSTORE-510)
+* Add isDcb flag in loan.json (CIRCSTORE-506)
+* Missing interface dependencies in module descriptor (CIRCSTORE-505)
```
Comment (roman-barannyk) [inline NEWS.md:12]:
```suggestion
* Add required interface `configuration` (CIRCSTORE-505)
```

File: NEWS.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
-## 17.3.0
-* [CIRCSTORE-533](https://folio-org.atlassian.net/browse/CIRCSTORE-533) Upgrade "holdings-storage" to 8.0
+## 17.3.0 2024-10-30
+* Upgrade RMB to 35.3.0, Vert.x to 4.5.9 (CIRCSTORE-520, CIRCSTORE-536)
+* Update module permissions in the ModuleDescriptor (CIRCSTORE-533)
+* Add checkout slips template (CIRCSTORE-529)
+* Setting name should be unique (CIRCSTORE-519)
```
Comment (roman-barannyk) [inline NEWS.md:5]: what changes are we announcing?

File: NEWS.md [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
+* Merge settings API into master branch (CIRCSTORE-515)
```
Comment (roman-barannyk) [inline NEWS.md:6]:
```suggestion
* Merge `circulation-settings` API into master branch (CIRCSTORE-515)
```

File: NEWS.md (hunk 1 above)
Comment (imerabishvili) [reply NEWS.md]: done

File: NEWS.md (hunk 3 above)
Comment (imerabishvili) [reply NEWS.md]: done

File: NEWS.md (hunk 4 above)
Comment (imerabishvili) [reply NEWS.md]: done

File: NEWS.md (hunk 5 above)
Comment (imerabishvili) [reply NEWS.md]: done

**Pull request merged**